### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by solomon-negusse**
see https://github.com/wri/gfw-data-api/pull/279 for details

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: app/models/orm/migrations/draft_versions/04fcb4f2408a_add_metadata_table.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
+"""add metadata table.
```
Comment (jterry64) [inline app/models/orm/migrations/draft_versions/04fcb4f2408a_add_metadata_table.py:1]: do we need this draft version one?

File: app/routes/datasets/queries.py [hunk 2]
```diff
@@ -646,25 +646,7 @@ def _get_default_layer(dataset, pixel_meaning):
...
-            ]
-        )
-        .where(
-            and_(
-                AssetORM.asset_type == AssetType.raster_tile_set,
-                VersionORM.is_latest == True,  # noqa: E712
-            )
-        )
-    ).gino.all()
+    latest_tile_sets = await assets.get_raster_tile_sets()
```
Comment (jterry64) [inline app/routes/datasets/queries.py:649]: did you test OTF at all on staging?

**Review by jterry64 — APPROVED** [review]
Nothing jumps out to me other than 2 small comments

File: app/routes/datasets/queries.py (hunk 2 above)
Comment (solomon-negusse) [reply app/routes/datasets/queries.py:649]: good call, I just did on a couple of geostores.. here is [one](staging-data-api.globalforestwatch.org/dataset/umd_glad_sentinel2_alerts/v20230202/query/json?geostore_id=46df30d9a44192efd90923af22e4fd0e&geostore_origin=rw&sql=select+latitude,+longitude,+umd_glad_sentinel2_alerts__date,+umd_glad_sentinel2_alerts__confidence+from+umd_glad_sentinel2_alerts+where+umd_glad_sentinel2_alerts__date+%3E+'2022-10-22'+and+umd_glad_sentinel2_alerts__date+%3C+'2023-04-20'+ORDER+BY+umd_glad_sentinel2_alerts__date+DESC+LIMIT+10000).

File: app/models/orm/migrations/draft_versions/04fcb4f2408a_add_metadata_table.py (hunk 1 above)
Comment (solomon-negusse) [reply app/models/orm/migrations/draft_versions/04fcb4f2408a_add_metadata_table.py:1]: I'll remove this. Created this temporarily to check in alembic scripts without doing the db change.

**Pull request merged**